orca-kotlin/src/main/kotlin/com/netflix/spinnaker/orca/ext/Stage.kt

```diff
@@ -103,34 +103,47 @@ fun Stage.allAfterStagesSuccessful(): Boolean =
 fun Stage.anyBeforeStagesFailed(): Boolean =
   beforeStages().any { it.status in listOf(TERMINAL, STOPPED, CANCELED) }
 
-fun Stage.anyAfterStagesFailed(): Boolean =
```

Comment:
Why was this removed?

Reply:
it wasn't used, so i figured I'd remove it

Reply:
will put back in